### ISSUE TITLE
Duplicate Item Problem

### ISSUE THREAD

**Issue by SirCraftypants** [RockinChaos/ItemJoin#543]
### Server Version

PaperSpigot 1.20.1

### Resource Version

ItemJoin-6.0.2-SNAPSHOT-b896

### Dump Information

https://pastebin.com/Wb3NzCMx

### Describe The Problem

I have a duplicate item problem where this Item is given twice, i've made sure no keep inventory is enabled anywhere, i've tried removing the items flags temporarily to destroy the duplicate items in lava but, it just keeps coming back. Driving me insane.

It returns everytime i rejoin the world in any free slot when only one should be appearing in slot 8.

### Checklist

- [X] I am running latest version of this resource.
- [X] I have read the wiki to make sure it's not an issue with configuration.
- [X] I ticked all of these boxes without actually reading them.
- [X] I have checked that my help request is unique and another request does not exist.
- [X] I acknowledge this is not a bug report and is instead a help request.

**Comment by SirCraftypants** [comment]
nevermind, dying to the void seems to have solved it.

**Comment by SirCraftypants** [comment]
Seems no matter how many times you die to the void, the item is duplicated a second time in the action bar despite it not being instructed too.

**Comment by SirCraftypants** [comment]
https://paste.helpch.at/mezubebuye.yaml

As you can see here it's telling your plugin to place an item in slot 8, 1 time. it's placing a duplicate item in slot 4 also.

**Comment by RockinChaos** [comment]
Hmmm, would you be able to do the following so I can further look into this;

1.) Try enabling debugging in the ItemJoin's config.yml and `/ij reload`. Then try to produce the duplication issue. How many times does it appear in the console window that you have been given the item?
2.) I need an `/itemjoin dump` URL so I can get a list of all your plugins. Preferably do this after the above step so I can also review the debug messages sent myself as well.

**Comment by SirCraftypants** [comment]
nevermind, was human error.  In case anyone else does the same i'll explain it here.

I created an item for a skyblock menu and had it generate in slot 4, then i added the flag `death-keep`, then i destroyed the item in slot 4 and set to the slot 8 instead.  As a result it was appearing in both slots all the time.

I removed the flag `death-keep` and died, now the slot 4 items gone for good and it only appears in slot 8. Problem Solved.
